x-pack/plugins/serverless_search/public/application/components/pipeline_panel.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiThemeProvider,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiText,
+  EuiLink,
+  EuiImage,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { LEARN_MORE_LABEL } from '../constants';
+
+export interface PipelinePanelProps {
+  assetBasePath: string;
+  docLinks: { elasticsearchClients: string; };
+}
+
+export const PipelinePanel: React.FC<PipelinePanelProps> = ({
+  assetBasePath,
```

Comment:
We don't need to pass down assetBasePath here. We're inside the Serverless Search plugin, so the basePath is known and always the same. We're passing it down as a prop elsewhere because that's in a shared package used by multiple plugins which each have a different base path, but that's not the case here.

Reply:
How do we will get the full length if we do not use `assetBasePath`? 
I tried with directory structure too
`<EuiImage alt="reporter" src={'../../assets/reporter.svg} />`

Reply:
There is already a hook `useAssetBasePath` exists.